src/Sinch/Verification/Report/Request/ReportPhoneCallVerificationRequest.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Sinch.Verification.Report.Request
 {
-    public class PhoneCallVerificationReportRequest : VerifyReportRequest
+    public class ReportPhoneCallVerificationRequest : VerifyReportRequest
```

Comment:
Various options for naming this one:
  - phone call
  - PSTN call
  - callout

  In the Node.js SDK, I've chosen `callout` because:
   1. it's the method name in the request body
   2. it's consistent with the Voice API which also uses this term

Reply:
yeah, makes sense